cmake/Doxygen.cmake

```diff
@@ -27,6 +27,7 @@ if(DOXYGEN_FOUND)
     set(ALL "ALL")
   endif()
   add_custom_target(doc ${ALL}
+                    COMMAND rm -rf ${CMAKE_BINARY_DIR}/doc/{html,xml}
```

Comment:
I don't believe this works, this doesn't go through a shell to expand the braces:

```plain
$ touch build-Debug/doc/{html,xml}/foo
$ cmake --build build-Debug --target doc
# …
$ ls build-Debug/doc/{html,xml}/foo
build-Debug/doc/html/foo  build-Debug/doc/xml/foo
```

Either way, shouldn't we be cleaning the whole `doc` folder?
```
$ tree -L1 build-Debug/doc   
build-Debug/doc
├── extra.css
├── footer.html
├── html
└── xml
```

Reply:
No, `extra.css` and `footer.html` are generated by `configure_file` and cleaning them here would cause problems.